WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -155,6 +166,27 @@ class WPRichContentView: UITextView {
         }
     }
 
+    private func setupTouchDetection() {
+          for gesture in gestureRecognizers ?? [] {
+              gesture.require(toFail: linkTapGestureRecognizer)
+          }
+          addGestureRecognizer(linkTapGestureRecognizer)
+      }
+
+      @objc func tapRecognized(_ recognizer: UIGestureRecognizer) {
+          let point = recognizer.location(in: self)
+          let characterIndex = self.layoutManager.characterIndex(for: point, in: self.textContainer, fractionOfDistanceBetweenInsertionPoints: nil)
+          // handle tap on link
+          if let linkAttribute = self.attributedText?.attribute(.link, at: characterIndex, effectiveRange: nil),
+              let url = linkAttribute as? URL,
```

Comment:
This was probably an existing issue, but I wanted to test links in comments and somehow the first comment with a link I could find was an email address. `mailto:user@example.com` is a URL, but we don't want our in-app browser to handle that (and it can't)

![Simulator Screen Shot - iPhone 11 - 2019-10-03 at 09 48 53](https://user-images.githubusercontent.com/8739/66108558-2265ef80-e5c3-11e9-9e50-7397d7a4fa04.png)

Reply:
Confirmed it was the same in latest WP Internal

Reply:
Right. Out of scope for this PR. I'll create a ticket for it if one doesn't exist

Reply:
A ticket exists: https://github.com/wordpress-mobile/WordPress-iOS/issues/11645